packages/hash/frontend/src/components/layout/PageHeader/AccountDropdown.tsx

```diff
@@ -74,7 +80,11 @@ export const AccountDropdown: VoidFunctionComponent<AccountDropdownProps> = ({
             ":hover": {
               boxShadow: "0px 0px 0px 2px #FFFFFF, 0px 0px 0px 5px #C1CFDE",
             },
-          }}
+            "&:focus-within": {
+              outline: "none",
+              boxShadow: `0px 0px 0px 2px #FFFFFF, 0px 0px 0px 5px ${palette.blue[70]}`,
+            },
+          })}
```

Comment:
Because you're reusing this styling in more than one place, I think this styling should be defined in a `styled` component instead.

Reply:
I think we should merge as is. I already started working on the focus states for IconButtons in this PR https://github.com/hashintel/hash/pull/409 and plan to complete it there. So we might end up having to remove the focus styles from here since I think that should be handled in `MuiIconButtonThemeOptions`. We can handle the hover styles in this component since it's different from other IconButtons on the page

(https://github.com/hashintel/hash/blob/a8fe740ae14ba2e6a5955352c09bdfd932d62ea3/packages/hash/frontend/src/theme/components/dataDisplay/MuiIconButtonThemeOptions.ts#L25-L32)

Reply:
Hey Ben, do you think abstracting the `sx` prop instead would be better? Given the styles in `IconButton` depend on the props of its parent.

Related documentation: https://mui.com/system/styled/#patterns-for-how-to-use-props-differ

Reply:
In an internal discussion agreed to try and define a `styled` component for this